src/website/views/custom-elements/button.pug

```diff
@@ -0,0 +1,301 @@
+
+h4 Flat
+p.-text
+  | To render a button with a transparent background and border, set the attribute <code>type</code> value to
+  | <code>flat</code> on the <code>&lt;chi-button&gt;</code> tag.
+.example.-mb--3
+  .-p--3.-d--flex
+    each type, val in {Base:'', Primary:'primary', Secondary:'secondary', Danger:'danger', Dark:'dark'}
+      div.-pr--2
+        chi-button(color=type, flat)
+          span=val
+  :code(lang='html')
+    <chi-button flat>Base</chi-button>
+    <chi-button color='primary' flat>Primary</chi-button>
+    <chi-button color='secondary' flat>Secondary</chi-button>
+    <chi-button color='danger' flat>Danger</chi-button>
+    <chi-button color='dark' flat>Dark</chi-button>
+
+h3 Pill
+p.-text To render buttons with a more pronounced border-radius, set the attribute <code>type</code> value to
+  | <code>pill</code> on the <code>&lt;chi-button&gt;</code> tag..
+h4 Base
+.example.-mb--3
+  .-p--3.-d--flex
+    each type, val in {Base:'', Primary:'primary', Secondary:'secondary', Danger:'danger', Dark:'dark'}
+      div.-pr--2
+        chi-button(color=type, type='pill')
+          span=val
+  :code(lang='html')
+    <chi-button type='pill'>Base</chi-button>
+    <chi-button color='primary' type='pill'>Primary</chi-button>
+    <chi-button color='secondary' type='pill'>Secondary</chi-button>
+    <chi-button color='danger' type='pill'>Danger</chi-button>
+    <chi-button color='dark' type='pill'>Dark</chi-button>
+
+h3 Fluid
+p.-text Buttons can be expanded to fill the parent space by applying the attribute
+  | <code>fluid</code> on the <code>&lt;chi-button&gt;</code> tag.
+.example.-mb--3
+  .-p--3
+    each type, val in {Base:'', Primary:'primary', Secondary:'secondary', Danger:'danger', Dark:'dark'}
+      div.-pt--2
+        chi-button(color=type, fluid)
+          span=val
+  :code(lang='html')
+    <chi-button fluid>Base</chi-button>
+    <chi-button color='primary' fluid>Primary</chi-button>
+    <chi-button color='secondary' fluid>Secondary</chi-button>
+    <chi-button color='danger' fluid>Danger</chi-button>
+    <chi-button color='dark' fluid>Dark</chi-button>
+
+h4 Outline
+.example.-mb--3
+  .-p--3.-d--flex
+    each type, val in {Base:'', Primary:'primary', Secondary:'secondary', Danger:'danger', Dark:'dark'}
+      div.-pr--2
+        chi-button(color=type, outline, type='pill')
+          span=val
+  :code(lang='html')
+    <chi-button type='pill' outline>Base</chi-button>
+    <chi-button color='primary' type='pill' outline>Primary</chi-button>
+    <chi-button color='secondary' type='pill' outline>Secondary</chi-button>
+    <chi-button color='danger' type='pill' outline>Danger</chi-button>
+    <chi-button color='dark' type='pill' outline>Dark</chi-button>
+
+h3 Icon buttons
+p.-text
+  | For interfaces with limited space, an <code>icon</code> button can
+  | be used for common actions such as editing, deleting, closing, etc.
+.example.-mb--3
+  .-p--3
+    chi-button(type='icon')
+      .a-icon
```

Comment:
Please remove all div.a-icon elements. Changing the display properties you will see in comments below will do the work. Change first that display properties and the remove these div.a-icon elements.